p2p/peer.go

```diff
@@ -256,15 +256,39 @@ func (p *peer) Status() cmtconn.ConnectionStatus {
 //
 // thread safe.
 func (p *peer) Send(e Envelope) bool {
-	return p.send(e.ChannelID, e.Message, p.mconn.Send)
+	start := time.Now()
```

Comment:
`cmttime`? I know it doesn't make any difference, as it's not used as wallclock time, but to calculate duration, but with `cmttime` the code is more homogeneous.

Reply:
We don't use `cmttime` in this file, but this is a good point. We lose the non-monotonic part, but this shouldn't be a problem.